### PR TITLE
Upgrade to Ruby 2.7.0

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

**Comment by huacnlee** [comment]
https://github.com/docker-library/official-images/blob/master/test/tests/ruby-binstubs/run.sh#L8

Need use `Bundler 2.1.2`

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -3,12 +3,12 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[2.6]='2 latest'
-	[2.7-rc]='rc'
+	[2.7]='rc'
```
Comment (eitoball) [inline generate-stackbrew-library.sh]: Shouldn't this be like following and remove line above (`[2.6]='2 latest'`)?

```suggestion
	[2.7]='2 latest'
```

File: generate-stackbrew-library.sh [hunk 2]
```diff
@@ -3,12 +3,12 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[2.6]='2 latest'
-	[2.7-rc]='rc'
+	[2.7]='2 latest'
```
Comment (ClaytonPassmore) [inline generate-stackbrew-library.sh]: I don't think you want both 2.6 and 2.7 aliased to `2 latest`. I'm guessing you can remove the 2.6 alias. 🙂

File: generate-stackbrew-library.sh [hunk 3]
```diff
@@ -3,12 +3,12 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[2.6]='2 latest'
```
Comment (ClaytonPassmore) [inline generate-stackbrew-library.sh]: I don't think you want both 2.6 and 2.7 aliased to 2 latest. I'm guessing you can remove the 2.6 alias. 🙂

**Comment by Shleeble** [comment]
Weird. ruby-binstubs still failing?

**Comment by JulienDefrance** [comment]
> Weird. ruby-binstubs still failing?

@huacnlee pointed out the version of bundler's incorrect.

**Comment by Shleeble** [comment]
Also, please add a stretch build for 2.7 now that it's stable.

**Comment by huacnlee** [comment]
@Shleeble I am sorry~,Because I don't know how to change the config files, do you mind to merge this PR first, and then add the `stretch`?

I submit this PR just want to get Ruby 2.7 Docker image as quick as possible

File: generate-stackbrew-library.sh (hunk 3 above)
Comment (tianon) [inline generate-stackbrew-library.sh]:
```suggestion
	[2.8-rc]='rc'
```

**Pull request merged**